gem/templates/rwanda/core/footer_page.html

```diff
@@ -0,0 +1,36 @@
+{% extends "base.html" %}
+{% load wagtailimages_tags %}
+{% block body_class %}template-{{ self.get_verbose_name|slugify }}{% endblock %}
+
+{% block content %}
+<div class="articles nav">
+  <h3 class="article-title">{{self.title}}</h3>
+  <p>{{self.subtitle}}</p>
+
+  {% if self.image %}
+  <div class="banner">{% image self.image width-320 %}</div>
+  {% endif %}
+
+<article>
+  {% for block in self.body %}
```

Comment:
this this a django template block instance? `{%block content %} {%endblock%}`
I.E. are we getting all the blocks in the template body and rendering them?
If not the word choice is a bit confusing

Reply:
@sewagodimo is currently a clone of the base template (initial setup) - will be fixed on subsequent tickets

Reply:
Updated